SECP-service/src/main/java/com/visucius/secp/models/Group.java

```diff
@@ -0,0 +1,69 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Table(name = "group")
+public class Group {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", unique = true, nullable = false)
+    private int id;
+
+    @Column(name = "name", unique = true, nullable = false)
+    private String name;
+
+    @Column(name = "visibility", nullable = false)
+    private String visibility;
+
+    @ManyToMany()
+    @JoinTable(name = "group_user",
+        joinColumns = { @JoinColumn(name = "group_id") },
+        inverseJoinColumns = { @JoinColumn(name = "user_id") })
+    private Set<User> users = new HashSet<>();
```

Comment:
please kindly test this relationship manually by creating some users and group in the db :)